python/GafferImageTest/ImageReaderTest.py

```diff
@@ -227,10 +227,12 @@ def assertMatch() :
 
 		reader["missingFrameMode"].setValue( GafferImage.OpenImageIOReader.MissingFrameMode.Black )
 		oiio["missingFrameMode"].setValue( GafferImage.OpenImageIOReader.MissingFrameMode.Black )
+		formatPlug = GafferImage.FormatPlug()
 		with context :
+			defaultFormat = formatPlug.getDefaultFormat(context)
 			self.assertRaisesRegexp( RuntimeError, ".*incompleteSequence.*.exr.*", reader["out"].image )
 			self.assertRaisesRegexp( RuntimeError, ".*incompleteSequence.*.exr.*", reader["out"]["format"].getValue )
-			self.assertEqual( reader["out"]["dataWindow"].getValue(), oiio["out"]["dataWindow"].getValue() )
```

Comment:
I don't think this part of the test should change. the `ImageReader` is still supposed to return the value it gets from the internal `OpenImageIOReader`